jbehave-support-core/src/main/java/org/jbehavesupport/core/web/WebSteps.java

```diff
@@ -216,7 +217,7 @@ public void navigateForward() {
 
     private String parseConditionValue(String condition) {
         String[] conditionParts = condition.split(" ");
-        return conditionParts.length == 3 ? conditionParts[2] : null;
+        return conditionParts.length > 3 ? String.join(" ", asList(conditionParts).subList(2,conditionParts.length)) : null;
```

Comment:
@AmyKosinova thank you for your thoughts and probably we should provide support for those  cases. I was however thinking about this and it seems to me that it is more complex than at first glance. We need to think of multiple scenarios here with HasAtrritube wait condition.
- **containing any** vs **containing all** - as I'm thinking of it from user point of view it should be probably containing all of attributes, but some cases could want any of attributes. How we should deal with it? possibly 2 separate WaitConditions.
- **ordering of conditions** (assuming we also checking value), if we go with containing all conditions in some cases like **class**, **style** etc order is not important but for others like **alt** ordering is important "image of cat" vs "cat of image".

Should we partially fix it just by checking any? Would it be expected behavior? Is there current requirement for this?
For that complexity first question here is, do we want to do that? And second if we do, possibly create new issue and discuss it in details in there.

Reply:
Hopefully there will be still space for more specific conditions in future: `... has any of class`, `...has all of classes` etc. Therefore we could probably stick with current solution, checking multi word expression inside html attributes.

Reply:
I have created issue #48 to solve this more specific conditions in case there is requirement and/or we have time to solve this.
Resolving this conversation.